spec/features/reservations/user_can_create_a_reservation_spec.rb

```diff
@@ -93,7 +93,7 @@
     expect(page).to_not have_content("Reservation ID:")
   end
 
-  it "cannot be created by a guest" do
+  xit "cannot be created by a guest" do
```

Comment:
The reason it's failing is because viewing a new reservation now has a partial on it. When I wrote this test, everything was passing, but no partials had been added for the sidebar. All you have to add is:

```
  <% if current_user %>
    <%= render partial: "shared/sidebar" %>
  <% end %>
```
to around the statement where `<%= render partial: "shared/sidebar" %>` is inside of the reservations/new template